src/components/definition-list/definition-list.stories.tsx

```diff
@@ -0,0 +1,189 @@
+import React from "react";
+import { Dl, Dt, Dd } from ".";
+import Icon from "../icon";
+
+import Box from "../box";
+import { ActionPopover, ActionPopoverItem } from "../action-popover";
+import Hr from "../hr";
+import Typography from "../typography";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+export const DefaultStory = () => (
+  <Dl>
+    <Dt>First</Dt>
+    <Dd>Description</Dd>
+    <Dt>Second</Dt>
+    <Dd>Description</Dd>
+    <Dt>Third</Dt>
+    <Dd>Description</Dd>
+  </Dl>
+);
+
+export const ActionPopoverAndIconSupport = () => (
+  <Dl>
+    <Dt>
+      <Box paddingTop="4px">Term example</Box>
+    </Dt>
+    <Dd>
+      <Box display="inline-flex" alignItems="center">
+        <Box mr={1}>Details example</Box>
+        <Icon type="tick" />
+      </Box>
+    </Dd>
+    <Dt>
+      <Box paddingTop="4px">Term example</Box>
+    </Dt>
+    <Dd>
+      <Box display="inline-flex" alignItems="center">
+        <Icon mr={1} type="tick" />
+        <Box mr={2}>Details example</Box>
+        <ActionPopover rightAlignMenu>
+          <ActionPopoverItem>Option 1</ActionPopoverItem>
+          <ActionPopoverItem>Option 2</ActionPopoverItem>
+        </ActionPopover>
+      </Box>
+    </Dd>
+  </Dl>
+);
+
+export const WithConditionalRendering = () => (
+  <Dl>
+    <Dt>First</Dt>
+    <Dd>Description</Dd>
+    <Dt>Second</Dt>
+    <Dd>Description</Dd>
+    {true && (
+      <>
+        <Dt>Third inside of React Fragment</Dt>
+        <Dd>Description inside of React Fragment</Dd>
+      </>
+    )}
+  </Dl>
+);
+
+WithConditionalRendering.parameters = { chromatic: { disableSnapshot: true } };
+
+export const AsASingleColumn = () => (
+  <Dl w={200} dtTextAlign="left" asSingleColumn>
+    <Dt>First</Dt>
+    <Dd>Description</Dd>
+    <Dt>Second</Dt>
+    <Dd>
+      <Box display="inline-flex" alignItems="center">
+        <Box mr={1}>Details example</Box>
+        <Icon type="tick" />
+      </Box>
+    </Dd>
+    <Dt>Third</Dt>
+    <Dd>Description</Dd>
+  </Dl>
+);
+
+export const MultipleSingleColumnsWithSegments = () => (
```

Comment:
**comment**: you should add `MultipleSingleColumnsWithSegments.parameters = { chromatic: { disableSnapshot: true } }` because the previous MDX version had this in the Story props.